.github/workflows/ci.yml

```diff
@@ -7,10 +7,15 @@ jobs:
       - uses: actions/checkout@v2
       - uses: olafurpg/setup-scala@v10
       - run: ./scalafmt --test
-  build:
+  template-tests:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - uses: olafurpg/setup-scala@v10
       - run: sbt test
-      - run: cd scalafix && sbt test
+  migration-rules:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v10
+      - run: find migration-rules/ -mindepth 1 -maxdepth 1 -type d -execdir sh -c 'cd {} && sbt test' \;
```

Comment:
wow, it's working! https://github.com/scalacenter/scalafix.g8/pull/20/checks?check_run_id=2602704306